spec/features/cart/unauthenticated_user_cart_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+feature "an unauthenticated user accesses a cart" do 
+
+  before(:each) do 
+  category = Category.create({name: "Lager",
+                                 description: "Yum"
+                                 })
+
+  beer1 = {name: "Titan",
+          state: true,
+          description: "aggressively hopped ipa",
+          price: 400}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.